k8s/spark-operator/schema.yaml

```diff
@@ -1,19 +1,19 @@
 application_api_version: v1beta1
 properties:
-  name:
+  APP_INSTANCE_NAME:
```

Comment:
This kind of change in the schema file makes the new version incompatible with the old version. If we do this, we should introduce a new version track and deprecate the old one.

Reply:
OK, you meant we need some change to the internal piece, right?

Reply:
Yea, we need to update the solution metadata.

Reply:
I updated the internal metadata and created a cl for you to review.